### DISCUSSION TITLE
Prevent users from changing Document Settings, Deleting Documents, or Deleting Sections/Containers in the Role Manager (Migrated from #18045)

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#27549] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Currently, elementor role management only disables the edit option with elementor. But security is still very weak. For example, a user can change the title of a template or page with the role of store manager and several other roles. They can even remove it!
This problem occurred to me on one of the customer websites. The client deleted the home screen.
I suggest adding a new option to the role manager. In a way that hides all the important options on the page and theme builder.
in page : https://prnt.sc/ctjq74_Ol3RX                    The only option that does not need to be hidden, is **view**.
in theme builder : https://prnt.sc/rVe-0i1-ikcT                   The only option that does not need to be hidden, is **view**.

These are really necessary to increase security and prevent many problems.

### Describe the solution you'd like

We can prevent the possibility of editing, changing the title and slug, outputting and also deleting posts (page & themplate) for different roles.

**Comment by trouaud** [comment]
+1
